tests/e2e/fixtures/formatted.cairo

```diff
@@ -0,0 +1,34 @@
+# Declare this file as a StarkNet contract and set the required
+# builtins.
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+
+# Define a storage variable.
+@storage_var
+func balance() -> (res : felt):
+end
+
+# Increases the balance by the given amount.
+@external
+func increase_balance{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    amount : felt
+):
+    let (res) = balance.read()
+    balance.write(res + amount)
+    return ()
+end
+
+# Returns the current balance.
+@view
+func get_balance{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+    res : felt
+):
+    let (res) = balance.read()
+    return (res)
+end
+
+@constructor
+func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
+    return ()
+end
```

Comment:
You can make this an integration test now. Check `ProtostarFixture`.

Reply:
I think we might want to keep both of them, it's core functionality after all.